testcafe/service.js

```diff
@@ -68,29 +67,30 @@ test.skip('Confirm Service Schedule Day Can Be Added', async t => {
 
   await modifySheduleTime(t, 'add');
 
+  await t.navigateTo(servicePage.url(serviceId));
+  await servicePage.waitUntilPageLoaded(t);
   await t
-    .navigateTo(editResourcePage.url(1))
     .expect(servicePage.schedule.count)
     .eql(originalScheduleDayCount + 1);
 });
 
 // TODO: Uncomment with the completion of ISSUE #594
 test.skip('Confirm Service Schedule Day Can Be Deleted', async t => {
```

Comment:
Can this test be un-skipped now too?

Reply:
Unfortunately, no. I tried uncommenting it, but it looks like that test is still broken for other reasons. I think #594 captures getting that bug fixed.